### PR TITLE
add Folia

### PR DESCRIPTION
**Pull request by Panda260**
# Description

I added the Folia project from PaperMC

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [X] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

## New egg Submissions

1. [X] Does your submission pass tests (server is connectable)?
2. [ ] Does your egg use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [X] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
4. [X] Have you added a unique README.md for the egg you are adding according to the alphabetical order?
5. [X] You verify that the start command applied does not use a shell script
    * [ ] If some script is needed then it is part of a current yolk or a PR to add one
6. [X] The egg was exported from the panel

### CODE REVIEW

**Review by Suqatri — APPROVED** [review]

**Comment by QuintenQVD0** [comment]
This egg is made or exported in a old version of the panel. Update the panel then export. Alsonudr the new 95% startup flag for´max ram for java

**Comment by Panda260** [comment]
> This egg is made or exported in a old version of the panel. Update the panel then export. Alsonudr the new 95% startup flag for´max ram for java

My panel in on the Latest version!
https://img.neverstopgaming.net/Panda/chrome_gGdSDmBIPm.png

**Comment by Panda260** [comment]
what did you mean with? "Alsonudr the new 95% startup flag for´max ram for java"

**Comment by Panda260** [comment]
you mean -XX:MaxRAMPercentage=95.0 ?

**Comment by QuintenQVD0** [comment]
> you mean -XX:MaxRAMPercentage=95.0 ?

yea ofc

**Comment by QuintenQVD0** [comment]
> > This egg is made or exported in a old version of the panel. Update the panel then export. Alsonudr the new 95% startup flag for´max ram for java
> 
> My panel in on the Latest version! https://img.neverstopgaming.net/Panda/chrome_gGdSDmBIPm.png

your egg export is old as it is still in v1 format so your panel is not up to date but you showed it is up to date so it should export in v2 format

**Comment by Panda260** [comment]
> > > This egg is made or exported in a old version of the panel. Update the panel then export. Alsonudr the new 95% startup flag for´max ram for java
> > 
> > 
> > My panel in on the Latest version! https://img.neverstopgaming.net/Panda/chrome_gGdSDmBIPm.png
> 
> your egg export is old as it is still in v1 format so your panel is not up to date but you showed it is up to date so it should export in v2 format

maybe because it is a copy of the Paper egg

**Comment by QuintenQVD0** [comment]
No certanly not! The v2 file is compeltly difrend. Just updatr your panel. And you can not directly edit the json file.

File: game_eggs/minecraft/java/folia/egg-folia.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-07-04T01:55:37+02:00",
```
Comment (QuintenQVD0) [inline game_eggs/minecraft/java/folia/egg-folia.json:7]: Import this egg in to a up to date panel and export it again and fully replace it here.

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by Panda260** [comment]
> No certanly not! The v2 file is compeltly difrend. Just updatr your panel. And you can not directly edit the json file.

the Panel says that it is up to date as you can see in the screenshot above
I have the version 1.11.3 is it the Latest one?

**Comment by QuintenQVD0** [comment]
> > No certanly not! The v2 file is compeltly difrend. Just updatr your panel. And you can not directly edit the json file.
> 
> the Panel says that it is up to date as you can see in the screenshot above I have the version 1.11.3 is it the Latest one?
> 
> https://img.neverstopgaming.net/Panda/chrome_vlXQMZhRIR.png

Now it is fine. Dont you want your own email on it? 
If someone can test this then this is fine for me.

**Comment by Panda260** [comment]
e-mail? no, everything is fine i think

**Comment by Panda260** [comment]
what do we have to wait for until the PR is accepted?

**Comment by QuintenQVD0** [comment]
Anyone external must confirm it is working and can join.

**Comment by Panda260** [comment]
ah okey

**Comment by Suqatri** [comment]
> Anyone external must confirm it is working and can join.
works for me

**Review by parkervcp — APPROVED** [review]
I am approving this due to it being an upgraded version of paper with multi-threading support.

File: game_eggs/minecraft/java/folia/egg-folia.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-07-04T20:45:35+02:00",
+    "name": "Folia",
+    "author": "parker@pterodactyl.io",
```
Comment (parkervcp) [inline game_eggs/minecraft/java/folia/egg-folia.json:9]: I am not the author of this egg.

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: game_eggs/minecraft/java/folia/egg-folia.json (hunk 2 above)
Comment (Panda260) [reply game_eggs/minecraft/java/folia/egg-folia.json]: updatet

**Review by parkervcp — APPROVED** [review]

**Review by QuintenQVD0 — APPROVED** [review]
LGTM! thank you!

**Pull request merged**